### PR TITLE
Upgrade to Consul 0.6

### PR DESCRIPTION
**Pull request by darylrobbins**
Updating consul and consul-web-ui to v0.6

### CODE REVIEW

File: Casks/consul-web-ui.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask :v1 => 'consul-web-ui' do
-  version '0.5.2'
-  sha256 'ad883aa52e1c0136ab1492bbcedad1210235f26d59719fb6de3ef6464f1ff3b1'
+  version '0.6.0'
+  sha256 '73c5e7ee50bb4a2efe56331d330e6d7dbf46335599c028344ccc4031c0c32eb0'
 
   # bintray.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/consul-web-ui.rb]: Needs to be changed to `# hashicorp.com is the official download host per the vendor homepage`

**Comment by darylrobbins** [comment]
Updated web ui comment to reflect new official source

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 3bf0ca82b70f854a0e51417769a3648338523f9c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by vitorgalvao** [comment]
The change was combining the last two commits.

**Pull request closed without merging**